### PR TITLE
Fixing a typo that breaks bools in translate tests

### PR DESCRIPTION
**Pull request by oelbert**
**Description**
There is a typo in the translate test class that causes crashes with bool test data

**How Has This Been Tested?**
Tests don't crash with this fix

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] New check tests, if applicable, are included

### CODE REVIEW

**Review by FlorianDeconinck — APPROVED** [review]
Oh dear, that's one burning hotfix

**Review by fmalatino — APPROVED** [review]
LGTM, once tests pass will merge

**Pull request merged**
